src/styles/types.ts

```diff
@@ -0,0 +1,23 @@
+export enum JustifyContent {
+  Start = 'flex-start',
+  End = 'flex-end',
+  Center = 'center',
+  SpaceBetween = 'space-between',
+  SpaceAround = 'space-around',
+  SpaceEvenly = 'space-evenly'
+}
+
+export enum AlignItems {
+  Start = 'flex-start',
+  End = 'flex-end',
+  Center = 'center',
+  Baseline = 'baseline',
+  Stretch = 'stretch'
+}
+
+export enum FlexDirection {
+  Row = 'row',
+  RowReverse = 'row-reverse',
+  Column = 'column',
+  ColumnReverse = 'column-reverse'
+}
```

Comment:
yyyy, don't like this approach.  
Could we ignore sonarcube rule for these values?